lib/mix/lib/mix/tasks/new.ex

```diff
@@ -408,6 +408,7 @@ defmodule Mix.Tasks.New do
   # here (which is why it is important to import them last).
   #
   #     import_config "#{Mix.env}.exs"
+  #
```

Comment:
We shouldn't have this line in this template.

Reply:
true